src/main/java/com/fdmgroup/forex/controllers/PortfolioController.java

```diff
@@ -0,0 +1,21 @@
+package com.fdmgroup.forex.controllers;
+
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.fdmgroup.forex.services.PortfolioService;
+
+@RestController
+@RequestMapping("/api/v1/portfolio")
+@CrossOrigin(origins = "*")
```

Comment:
It would depend on the use-case. Allowing cross-origins from anywhere would potentially mean that anyone online could request data via that API (though in our case with authentication in place this ...should... be prevented. In practice, it would still be better to define the IP addresses that are permitted to access the methods.

In an ecommerce example, you might have GET methods for all product details, and GET methods for all invoices. It would be fine to permit public access to product data that is available on the ecommerce site anyway, but invoices should not be public. So if both were served by the same `controller`, you could specify that the product details mapping has `@CrossOrigin(origins = "*")`, but for the invoice GET request restrict the CORS to only those you recognise.